### PR TITLE
Feature/1495/verification

### PR DESCRIPTION
**Pull request by garyluu**
For issue ga4gh/dockstore#1495

remove whitespace changes using "?w=1" in url or via github controls

First pass at using the new verificationSource object in sourcefiles.

3 changes:
- Version component removed tool/workflow path, verified column changed to verified platforms
- View version component has a verified table appended to it with (path, metadata, and platform fields)
- entry component sidebar updated to only show 1 version's verified info.

Also fixes the infamous starring bug

Also fixes the TRS link's href

### CODE REVIEW

**Comment by denis-yuen** [comment]
Remind me to review in action

**Comment by denis-yuen** [comment]
(suggestions provided in 1-1, request re-review when ready)

File: src/app/container/versions/versions.component.html [hunk 1]
```diff
@@ -101,13 +96,6 @@
           </span>
           <ng-template #noCommitID><span>{{ version.reference || 'n/a' }}</span></ng-template>
         </td>
-        <td *ngIf="tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
```
Comment (denis-yuen) [inline src/app/container/versions/versions.component.html]: Deleted on purpose?

File: src/app/shared/entry/verified-display/verified-display.component.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <table mat-table [dataSource]="dataSource" class="mat-elevation-z4" style="width: 100%" matSort>
+      <ng-container matColumnDef="path">
+        <mat-header-cell *matHeaderCellDef mat-sort-header> Test Parameter File </mat-header-cell>
+        <mat-cell  *matCellDef="let element"> {{element.path}} </mat-cell>
+      </ng-container>
+      <ng-container matColumnDef="platform">
+        <mat-header-cell *matHeaderCellDef mat-sort-header> Platform </mat-header-cell>
+        <mat-cell *matCellDef="let element"> {{element.platform}} </mat-cell>
+      </ng-container>
+      <ng-container matColumnDef="metadata">
```
Comment (denis-yuen) [inline src/app/shared/entry/verified-display/verified-display.component.html]: I think I prefer `Verifier` from the specification

File: src/app/container/versions/versions.component.html (hunk 1 above)
Comment (garyluu) [reply src/app/container/versions/versions.component.html]: Ya, I was under the impression we're deleting the "Paths" column from both tool and workflow version table

File: src/app/shared/entry/verified-display/verified-display.component.html (hunk 2 above)
Comment (garyluu) [reply src/app/shared/entry/verified-display/verified-display.component.html]: "metadata" is not user-facing, but I'll change it anyways for consistency.

Comment (denis-yuen) [reply src/app/shared/entry/verified-display/verified-display.component.html]: Oh, I meant `Verified Source` => `Verifier` but its good to keep the two consistent

**Pull request merged**